cloud/jenkins/pxc_operator_gke_version.groovy

```diff
@@ -302,23 +302,42 @@ pipeline {
                     }
                 }
                 stage('E2E Basic Tests') {
-                    steps {
-                        CreateCluster('basic')
-                        runTest('init-deploy', 'basic')
-                        runTest('limits', 'basic')
-                        runTest('monitoring-2-0', 'basic')
-                        runTest('affinity', 'basic')
-                        runTest('one-pod', 'basic')
-                        runTest('auto-tuning', 'basic')
-                        runTest('proxysql-sidecar-res-limits', 'basic')
-                        runTest('users', 'basic')
-                        runTest('haproxy', 'basic')
-                        runTest('tls-issue-self', 'basic')
-                        runTest('tls-issue-cert-manager', 'basic')
-                        runTest('tls-issue-cert-manager-ref', 'basic')
-                        runTest('validation-hook', 'basic')
-                        ShutdownCluster('basic')
-                   }
+                    stages {
+                        stage('cluster creation') {
+                            steps {
+                                CreateCluster('basic')
+                            }
+                        }
+                        stage('default CR') {
+                            when { expression { params.GIT_BRANCH.contains('release-') }}
```

Comment:
maybe it is better to add this logic into `runTest` function

Reply:
I've created a wrapper function. Please check